package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I wonder if this should be inside the ccd-case-generator module?

Reply:
My thinking too, I think this can be moved inside